plugins/traction_innkeeper/traction_innkeeper/v1_0/innkeeper/utils.py

```diff
@@ -45,6 +45,11 @@ class TenantConfigSchema(OpenAPISchema):
         ),
         description="Public DID config",
     )
+    self_issuer_permission = fields.Bool(
+        required=False,
+        description="True if tenant can make itself issuer, false if only innkeeper can",
+        default=False,
```

Comment:
Might be better described as `True if tenants are automatically allowed to be issuers, False if innkeeper approval is required". Maybe the flag needs to be renamed as well to something more consistent to the behaviour, like `auto_issuer_permission`?

Reply:
Changed to `auto_issuer` to be consistent with other flags in reservation config like `auto_approve`.